src/edge/templates/edge/edge.html

```diff
@@ -22,6 +22,25 @@
     <script src="{% static 'edge/assets/edge_jst.js' %}"></script>
     <script src="{% static 'edge/assets/edge.js' %}"></script>
     <link rel="stylesheet" href="{% static 'edge/assets/edge.css' %}">
+    <script>
+    window['_fs_debug'] = false;
+    window['_fs_host'] = 'fullstory.com';
+    window['_fs_script'] = 'fullstory.com/s/fs.js';
+    window['_fs_org'] = '{{ FULLSTORY_ORG_ID }}';
+    window['_fs_namespace'] = 'FS';
+    (function(m,n,e,t,l,o,g,y){
```

Comment:
It might be nice to wrap this function in a `if (window['_fs_org'])` so that it's usable without fullstory (since it's open source and might be used in that way).

Reply:
This is copied and pasted from fullstory.